browser/src/control/Control.ContextToolbar.ts

```diff
@@ -0,0 +1,296 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * L.control.ContextToolbar - definition of context toolbar
+ * This toolbar should appear over selection when related selection is done using mouse
+ */
+
+class ContextToolbar {
+	container: HTMLElement;
+	builder: any;
+	map: any;
+	initialized: boolean = false;
+	// roughly twice the height(76px) of default context toolbar in each direction from boundary
+	disappearingBoundary: number = 150; // px
+
+	constructor(map: any) {
+		this.map = map;
+		this.builder = new L.control.notebookbarBuilder({
+			windowId: -2,
+			mobileWizard: this,
+			map: this.map,
+			cssClass: 'notebookbar',
+			useSetTabs: true,
```

Comment:
Do not add this property (https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Definitions.Types.ts#L36) to avoid any problems.

It is used to put tabs using component's api but your ContextToolbar doesn't have these methods and are not needed (no tabs) so no sense to declare we support it.